### PR TITLE
update README with a non-solution statement

### PR DESCRIPTION
**Pull request by JaydenLiang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,10 @@
```
Comment (jamie-pate) [inline README.md:2]: # Autoscale Core Library

Adding 'library' to the title should help people notice that this is a library

File: README.md [hunk 2]
```diff
@@ -2,7 +2,10 @@
 
 [![version on main branch](https://img.shields.io/github/package-json/v/fortinet/autoscale-core?label=version%20on%20main%20branch)](./) [![latest release version](https://img.shields.io/github/v/release/fortinet/autoscale-core?label=latest%20release%20version)](https://github.com/fortinet/autoscale-core/releases/latest) [![platform](https://img.shields.io/badge/platform-AWS%20|%20Azure-green.svg)](./)
 
-Autoscale Core is the core library for the Fortinet Autoscale project.
```
Comment (jamie-pate) [inline README.md:5]: People probably will skim over just the first few words in the sentence. Better to have a note in the first paragraph that strongly grabs their attention.

Examples:
https://giters.com/gluck/node-git-rest-api
https://sqlite.org/src4/doc/trunk/www/index.wiki

Recommendation:
```Markdown
## This is probably not the project you are looking for

This project does not directly create a cloud deployment. Autoscale Core is the core library for the Fortinet Autoscale projects. To find complete solutions, please check the following projects:

* [FortiGate Autoscale for AWS](https://github.com/fortinet/fortigate-autoscale-aws)
* [FortiGate Autoscale for Azure](https://github.com/fortinet/fortigate-autoscale-azure)
* [FortiGate Autoscale for GCP](https://github.com/fortinet/fortigate-autoscale-gcp)
```

**Review by jamie-pate — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (jamie-pate) [reply README.md:5]: * People will likely skim over the entire text and **only read** the first few words of each sentence...

File: README.md (hunk 1 above)
Comment (JaydenLiang) [reply README.md:2]: added.

File: README.md (hunk 2 above)
Comment (JaydenLiang) [reply README.md:5]: added.

**Review by jamie-pate — APPROVED** [review]

**Pull request merged**